libshaderc_spvc/src/spvc_c_smoke_test.c

```diff
@@ -83,6 +86,9 @@ int main() {
     printf("failed to produce hlsl\n");
```

Comment:
Does this also need to release?

Reply:
*need* to, no because the test has failed.  I'd just as soon not complicate it, even if that is poor form.

Reply:
Well, it still leaks in this case, yea? Which means if we write a test which fails this on purpose we're going to get an ASAN leak bug from it aren't we?

Reply:
I don't see how that's possible, this test has no input or options and is hard coded to pass, but I'll add the free's, it's good form.